packages/graphql/src/translate/queryAST/factory/FilterFactory.ts

```diff
@@ -194,9 +141,15 @@ export class FilterFactory {
             const connectionWhereField = parseConnectionWhereFields(key);
             if (connectionWhereField.fieldName === "edge") {
                 console.log(connectionWhereField.fieldName, value);
+                const targetEdgeFilters = this.createEdgeFilters(relationship, value as any);
```

Comment:
We can remove that console log that I left :sweat_smile: 

Also, is it possible to avoid any here?